app/Http/Controllers/Users/UsersController.php

```diff
@@ -541,11 +543,74 @@ public function getExportUserCsv()
     {
         $this->authorize('view', User::class);
         \Debugbar::disable();
-
         $response = new StreamedResponse(function () {
+        $user = Auth::user();
+            $settings = Setting::getSettings();
             // Open output stream
             $handle = fopen('php://output', 'w');
 
+            if($settings->full_multiple_companies_support){
+                User::with('assets', 'accessories', 'consumables', 'department', 'licenses', 'manager', 'groups', 'userloc', 'company')
+                    ->where('company_id', '=', $user->company_id)
```

Comment:
Hm, we should probably respect the superadmin permissions here though. If I'm a superadmin, I should be able to download all, no?